recipes/pedantic/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "pedantic" %}
+{% set version = "1.14.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LostInDarkMath/pedantic-python-decorators/archive/refs/tags/20230913095009-ca2b278.tar.gz
```

Comment:
The fact that the tag doesn't have the specified version above will make this quite hard for the bot.

Also, the name is quite close to a very popular library. I wonder if PyPI will ever allow this to the uploaded b/c they are trying to protect users from naming confusion. In conda-forge we never discussed that but I wonder if we should start.

Reply:
Actualy there is one on PyPI! We should use that one to avoid the version issue in the tag.